pages/filters/create.vue

```diff
@@ -32,15 +32,16 @@
       <NotificationSettingsFilter :filter="filter"></NotificationSettingsFilter>
 
       <ConfirmCreation :is-new-filter="isNewFilter" />
+
     </form>
   </UiDefaultContainer>
 
+  <GeneralMatchTable :matches />
 </template>
 
 <script setup lang="ts">
-import NotificationSettingsFilter from '~/components/filters/NotificationSettingsFilter.vue';
-import CreateHeader from './Create/CreateHeader.vue';
-import ConfirmCreation from './Create/ConfirmCreation.vue';
```

Comment:
These need to be imported

Reply:
one of the main things about nuxt is the auto import feature
you basically never import components or vue things
https://nuxt.com/docs/guide/concepts/auto-imports

i can revert that part, but that should never be necessary and breaks conventions

Reply:
Nuxt by default only auto imports files in the components directory, those files are page specific and not global so nuxt doesn't auto import them. If you don't believe that is standard practice we can move those files into the components directory.